src/bing/aanmeldformulier/templates/aanmeldformulier/info.html

```diff
@@ -0,0 +1,60 @@
+{% extends "aanmeldformulier/base.html" %}
+{% load sniplates %}
+
+{% block page-content %}
+
+<div>
+
```

Comment:
Small thing there are newlines here that do not show in the other blocks of code.